enigma-swing/src/main/java/cuchaz/enigma/gui/util/GuiUtil.java

```diff
@@ -53,6 +54,24 @@ public static JLabel unboldLabel(JLabel label) {
         return label;
     }
 
+    /**
+     * Puts the provided {@code text} in the system clipboard.
+     */
+    public static void copyToClipboard(String text) {
+        Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(text), null);
+    }
+
+    public static void showPopup(JComponent component, String text, int x, int y) {
```

Comment:
Personally I'd put that message in the status bar but this works :P

Reply:
I think this is too far away from the copied text, I don't think people would notice it